src/routes/buses.router.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express"
+import { createBus, deleteBusById, getAllBuses, getBusById, updateBusById } from "../controllers/buses.controller"
```

Comment:
Can we please have a comma at the end of updateBusById so that the code may look cleaner?